extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -113,15 +113,21 @@ public function getMonetizationUnits( $params ) {
 		$cacheKey = $this->getMemcKey( $params );
 		$log->debug( "MonetizationModule: lookup with cache key: $cacheKey", $loggingParams );
 
-		$json_results = $this->wg->Memc->get( $cacheKey );
-		if ( !empty( $json_results ) ) {
-			$log->info( "MonetizationModule: memcache hit.", $loggingParams );
-			wfProfileOut( __METHOD__ );
-			return $this->setThemeSettings( $json_results, $cacheKey );
+		if ( $this->isPageSpecificRequest( $params ) ) {
+			$log->info( "MonetizationModule: page specific request.", $loggingParams );
+		} else {
```

Comment:
This is a matter of preference. Normally the condition that leads to a the early exit is first and the else block is executed and continues with the rest of the method.  Makes reading the code flow better IMO. 

Another way is to avoid early exists where possible. Not saying that is possible in this case tho.